StoreReviewHelper.swift

```diff
@@ -43,7 +43,7 @@ struct StoreReviewHelper {
     func requestReview() {
         if #available(iOS 14.0, *) {
             if let scene = UIApplication.shared.currentScene {
-                SKStoreReviewController.requestReview(in: scene)
+                //SKStoreReviewController.requestReview(in: scene)
```

Comment:
Commented lines should be uncommitted when committing.
```suggestion
                SKStoreReviewController.requestReview(in: scene)
```